raiden/encoding/messages.py

```diff
@@ -69,6 +68,7 @@ def make_message(message, **attrs):
 target = make_field('target', 20, '20s')
 initiator = make_field('initiator', 20, '20s')
 sender = make_field('sender', 20, '20s')
+channel = make_field('channel', 20, '20s')
```

Comment:
Question to make sure I properly comprehend the changes.
You have added the `channel` to all messages but it is not being consumed anywhere.

It seems to be consumed indirectly though through the hash of the first 65 bytes of the transfer. That would make messages unique per channel and would make the nonce ranges check for "replay attacks" irrelevant.

Correct?